api-docs/rst/dev-guide/concepts.rst

```diff
@@ -111,6 +111,16 @@ The launch configuration specifies the launch type along with server and load ba
              Not used when there is no ``loadbalancers`` configuration. Please note that
              this feature only works with a cloud load balancer.
 
+Each scaling group has an associated ``status`` that represents the health of the
+group. When the group is successfully able to launch servers and optionally add
+them to load balancers then the status is ``ACTIVE``. If it is not able to
+that due to some irrecoverable error that requires user attention then status
+will be ``ERROR``. In this case, the :ref:`group state <get-scaling-group-state>`
+will contain list of human-readable texts describing the reasons why group has
+ERROR status. These errors will need to be fixed before group can become ACTIVE
+again that can be done by either :ref:`converging <trigger-convergence>` or
+:ref:`executing policy <execute-policy>` after fixing the errors.
+
 
```

Comment:
If the scaling group cannot launch servers because of an error that requires user attention, 
the status changes to ``ERROR``.  In this case, the :ref:`group state <get-get-scaling-group-state-v1.0-tenantid-groups-groupid-state>` contains a list of human-readable messages that explain the conditions that caused the error. 
After you fix the errors, you can restore the group to ``ACTIVE`` state by submitting a 
 :ref:`trigger convergence <post-converge-v1.0-tenantid-groups-groupid>`  or
:ref:`execute policy <post-execute-policy-v1.0-tenantid-groups-groupid-policies-policyid-execute>` API request.
+